src/reducers/cdps.js

```diff
@@ -149,6 +150,50 @@ export function getDaiAvailable(cdp, rounded = true, precision = 2) {
       );
 }
 
+export function getEventHistory(cdp) {
```

Comment:
@tyler17 no need to change this now, but i figured it was still worth mentioning for learning purposes: it's best to create a mock like this as an async function, since the real data fetching that eventually replaces it will definitely be async. Then you can work out in the calling code how to handle that async-ness early on, so it's more of a drop-in replacement when the real data fetching is ready.

Reply:
👍